### DISCUSSION TITLE
Is Merge Confidence Beta disabled on PR grouping?

### DISCUSSION THREAD

**Discussion by MojoJojo86** [renovatebot/renovate#22912] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Latest 35.139.1

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

When grouping dependencies together such as 
```{
  "packageRules": [
    {
      "groupName": "all non-major dependencies",
      "groupSlug": "all-minor-patch",
      "matchPackagePatterns": [
        "*"
      ],
      "matchUpdateTypes": [
        "minor",
        "patch"
      ]
    }
  ]
}
```

The merge confidence beta disappears and I would have expected this to display per dependency that was due to be upgraded in the PR.

### What you tried so far.

Removing the grouping and the merge confidence returns on single PR's

**Comment by nabeelsaabna** [comment]
No, it should work also for grouped PRs
the [Merge Confidence Beta](https://github.com/whitesource/merge-confidence/blob/main/beta.json) is only applied for:
```
{
 "datasources": ["maven", "npm", "pypi"],
 "updateTypes": ["patch", "minor", "major"],
}
```

**Reply by MojoJojo86** [reply]
Hmmm doesn't seem to be, i'll do some more investigation to make sure there isn't anything contradicting this but I think single PR's might be the way to go anyway for naming conventions etc.

**Comment by MojoJojo86** [comment]
Need to investigate further.

**Reply by christopherbox** [reply]
Did you find a resolution to this? This seemingly is also occurring for me. Something is causing it to not appear.
